src/Molecules/TableList/SortedColumn/SortedColumn.jsx

```diff
@@ -104,7 +104,10 @@ const SortedColumn = ({
           setFilterAnchorEl(null)
           onFilter(type in EMPTY_VALUES ? EMPTY_VALUES[type] : '')
         }}
-        onChange={onFilter}
+        onChange={value => {
+          setFilterAnchorEl(null)

```

Comment:
In the case of `DateRange`, this will make the filter popover close when I try to change the end date, after changing the start date

Reply:
Yeaaah...this whole blur thing starts to bother me